### PR TITLE
Added mirror for PCRE because the two existing links did not work for me 

### PR DESCRIPTION
**Pull request by pk2061**
Hi,

I created a new mirror for the PCRE to https://ftp.pcre.org/pub/pcre/pcre-8.39.tar.bz2 because the existing links seemed to be broken.

### CODE REVIEW

File: Library/Formula/popt.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Popt < Formula
   desc "Library like getopt(3) with a number of enhancements"
   homepage "http://rpm5.org"
-  url "http://rpm5.org/files/popt/popt-1.16.tar.gz"
+  url "https://fossies.org/linux/misc/popt-1.16.tar.gz"
```
Comment (mistydemeo) [inline Library/Formula/popt.rb:4]: Looks like this is a 404.

**Review by mistydemeo — CHANGES_REQUESTED** [review]

File: Library/Formula/popt.rb (hunk 1 above)
Comment (pk2061) [reply Library/Formula/popt.rb:4]: Sorry, I did not mean to upload that as is.
I will take care of that.. :-)

Comment (pk2061) [reply Library/Formula/popt.rb:4]: Maybe just discard the changes to the popt.rb as here the original url http://rpm5.org/files/popt/popt-1.16.tar.gz seems to work just fine.
Sorry for the mess-up on my end here!

Comment (mistydemeo) [reply Library/Formula/popt.rb:4]: No worries at all!

**Comment by mistydemeo** [comment]
Merged; thank you very much for your contribution!

**Comment by cozycactus** [comment]
pcre links still broken here...

**Comment by mistydemeo** [comment]
This mirror worked when this PR was merged, but has broken since. I'll find a new source.

**Pull request closed without merging**